src/js/edu-benefits/containers/BenefitsSelection.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+
+import BenefitsSelectionFields from '../components/BenefitsSelectionFields';
+import NavButtons from '../components/NavButtons';
+import { isValidSection } from '../utils/validations';
+import { withRouter } from 'react-router';
+import { veteranUpdateField, ensureFieldsInitialized, updateCompletedStatus } from '../actions/index';
+
+class BenefitsSelection extends React.Component {
+  render() {
+    const { section, currentLocation, data, submission, router, onStateChange, dirtyFields, setComplete } = this.props;
+    const navigateTo = path => router.push(path);
+
+    return (
+      <div className="form-panel">
+        <BenefitsSelectionFields data={data} section={section} onStateChange={onStateChange}/>
```

Comment:
It seemed like we would need it on the review page, when you can edit the fields, but don't have the same navigation buttons.